packages/docusaurus-plugin-content-docs/src/theme/DocItem/index.js

```diff
@@ -53,7 +52,11 @@ function DocItem(props) {
             <DocPaginator docsMetadata={docsMetadata} metadata={metadata} />
           </div>
           <div className="col col--3 col--offset-1">
-            {DocContent.rightToc && <Headings headings={DocContent.rightToc} />}
+            {DocContent.rightToc && (
+              <div className={styles.tableOfContents}>
```

Comment:
Can we possibly pass the class name to `Headings` and reduce the wrapped DOM here?

Reply:
https://github.com/facebook/Docusaurus/blob/50f997b138d488e047b6760289816b0c0c999983/packages/docusaurus-plugin-content-docs/src/theme/DocItem/index.js#L15-L29

Headings is being used recursively.

Reply:
It's not good to pass classNames around as there can be undesired consequences (like changing the `display to flex) and then affecting all the children). Wrapping is better.